llvm/lib/Target/LoongArch/LoongArchISelLowering.cpp

```diff
@@ -1819,6 +1818,10 @@ SDValue LoongArchTargetLowering::getStaticTLSAddr(GlobalAddressSDNode *N,
   SDValue Offset = Large
                        ? SDValue(DAG.getMachineNode(Opc, DL, Ty, Tmp, Addr), 0)
                        : SDValue(DAG.getMachineNode(Opc, DL, Ty, Addr), 0);
+
+  if (IsNormalOrMediumLE)
```

Comment:
```suggestion
  if (Opc == LoongArch::PseudoLA_TLS_LE && !Large)
```

Reply:
It's better to add a comment here. If it is LE, the `add tp` operation will occur during the pseudo-instruction expansion.

Reply:
Thanks. I will add this.